<ui version="4.0" >
 <class>VideoOptions</class>
 <widget class="QWidget" name="VideoOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Video Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel" >
     <property name="text" >
      <string>Tit&amp;le:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>titleEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleEdit" />
   </item>
   <item>
    <widget class="QLabel" name="directoryLabel" >
     <property name="text" >
      <string>Custom &amp;preview:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>previewUrl</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="previewUrl" />
   </item>
   <item>
    <widget class="QLabel" name="aspectLabel" >
     <property name="text" >
      <string>&amp;Aspect ratio:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>aspectComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="aspectComboBox" >
     <item>
      <property name="text" >
       <string>4:3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>16:9</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="audioLabel" >
     <property name="text" >
      <string>Audio Trac&amp;ks:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>audioListBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="audioListBox" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="KPushButton" name="audioPropertiesButton" >
         <property name="text" >
          <string>Properties</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel" >
     <property name="text" >
      <string>Subtitles:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="subtitleListBox" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="KPushButton" name="subtitleAddButton" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="subtitleRemoveButton" >
         <property name="text" >
          <string>Remove</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="subtitlePropertiesButton" >
         <property name="text" >
          <string>Properties</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="chapterLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>xx</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="chapterPropertiesButton" >
       <property name="text" >
        <string>Properties</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>previewUrl</tabstop>
  <tabstop>aspectComboBox</tabstop>
  <tabstop>audioListBox</tabstop>
  <tabstop>audioPropertiesButton</tabstop>
  <tabstop>subtitleListBox</tabstop>
  <tabstop>subtitleAddButton</tabstop>
  <tabstop>subtitleRemoveButton</tabstop>
  <tabstop>subtitlePropertiesButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
